21torr/hosting/3.2/config/routes/hosting.yaml

```
@@ -0,0 +1,3 @@
+_import.hosting:
```

I think according to current bundle name it should be plain
```suggestion
hosting:
```

---

The keys of import routes are removed, right? Otherwise we might have an issue with name collisions.